ccpp/data/GFS_typedefs.F90

```diff
@@ -2554,6 +2555,35 @@ subroutine coupling_create (Coupling, IM, Model)
 !     Coupling%zorlwav_cpl  = clear_val
 !   endif
 
+    if (Model%cplflx .or. Model%cpllnd) then
```

Comment:
This is incredibly minor, but the style in the rest of this routine is to group all allocates together, then group all initializations together. Of course this is equivalent, but doesn't fit stylistically. Grouping allocations together and inits together certainly makes it faster to edit either operation in the future, rather than alternating allocation/initialization as done here.

Reply:
It is done with [877b29d](https://github.com/NOAA-EMC/fv3atm/pull/591/commits/877b29d2179653e5cac217d2b2a9ceb335566468). Thanks. It is always better to follow the conventions.